src/ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ASPNetCoreMastersTodoList.Api.BindingModels;
+using Microsoft.AspNetCore.Mvc;
+using Services;
+using Services.DTO;
+
+namespace ASPNetCoreMastersTodoList.Api.Controllers
+{
+    public class ItemsController : ControllerBase
+    {
+        public int Get(int id)
+        {
+            return new ItemService().Get(id);
+        }
+
+        public void Post(ItemCreateBindingModel itemCreateBindingModel)
```

Comment:
Need to add [FromBody] attribute for model binding. We also need to check the ModelState.IsValid to see model validation result

Reply:
This is noted. Have committed an update regarding this comment. Thank you!